service-front/public/css/bootstrap.css

```diff
@@ -0,0 +1,10878 @@
+@charset "UTF-8";
```

Comment:
Not sure if this CSS file is actually referenced in the demo app. Since we'll be using GDS styles it can probably just be removed and we can go styling free until we bring in the GDS styles